content/es/docs/concepts/services-networking/network-policies.md

```diff
@@ -9,82 +9,82 @@ weight: 50
+Las políticas de red no entran en conflicto; son aditivas. Si alguna política(s) se aplica a un Pod para una dirección determinada, las conexiones permitidas en esa dirección desde ese Pod son la unión de lo que permiten las políticas aplicables. Por lo tanto, el orden de evaluación no afecta al resultado de la política.
 
 Para que se permita una conexión desde un Pod de origen a un Pod de destino, tanto la política de salida del Pod de origen como la de entrada del Pod de destino deben permitir la conexión. Si cualquiera de los dos lados no permite la conexión, ésta no se producirá.
 
 
 ## El Recurso NetworkPolicy {#networkpolicy-resource}
 
-Ver la referencia [NetworkPolicy](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#networkpolicy-v1-networking-k8s-io) para una definición completa del recurso.
+Ver la referencia [NetworkPolicy](/docs/reference/generated/kubernetes-api/{{<param "version">}}/#networkpolicy-v1-networking-k8s-io) para una definición completa del recurso.
 
-Un ejemplo de NetworkPolicy pudiera ser este:
+Un ejemplo de NetworkPolicy podría ser este:
 
 {{% codenew file="service/networking/networkpolicy.yaml" %}}
 
-{{< note >}}
-Enviar esto al API Server de su clúster no tendrá ningún efecto a menos que su solución de red soporte de políticas de red.
-{{< /note >}}
+{{<note>}}
+Enviar esto al API Server de tu clúster no tendrá ningún efecto a menos que tu solución de red soporte políticas de red.
+{{</note>}}
 
-__Campos Obligatorios__: Como con todos los otras configuraciones de Kubernetes, una NetworkPolicy
+__Campos Obligatorios__: Como con todas las otras configuraciones de Kubernetes, una NetworkPolicy
 necesita los campos `apiVersion`, `kind`, y `metadata`.  Para obtener información general
-sobre cómo funcionan esos ficheros de configuración, puedes consultar
+sobre cómo funcionan esos archivos de configuración, puedes consultar
 [Configurar un Pod para usar un ConfigMap](/docs/tasks/configure-pod-container/configure-pod-configmap/),
 y [Gestión de Objetos](/docs/concepts/overview/working-with-objects/object-management).
 
 __spec__: NetworkPolicy [spec](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#spec-and-status) contiene toda la información necesaria para definir una política de red dado un Namespace.
 
-__podSelector__: Cada NetworkPolicy incluye un `podSelector` el cual selecciona el grupo de Pods en los cuales aplica la política. La política de ejemplo selecciona Pods con el label "role=db". Un `podSelector` vacío selecciona todos los Pods en un Namespace.
+__podSelector__: Cada NetworkPolicy incluye un `podSelector` el cual selecciona el grupo de Pods en los cuales aplica la política. La política de ejemplo selecciona Pods con la etiqueta "role=db". Un `podSelector` vacío selecciona todos los Pods en un Namespace.
```

Comment:
@electrocucaracha que es mejor dejarlo con label o como etiqueta, por lo que veo q estan las dos formar en mi PR

Reply:
Me parece que en algún momento se [decidió](https://docs.google.com/spreadsheets/d/1u42ssjDQTqa_YRLnVlLrcgKbGfei6adA--7IOHLJ2IM/edit#gid=1830888365) dejarlo como etiqueta, más nunca se mantuvo esa regla

Reply:
Por lo que veo en el documento que me adjuntas tiene las dos definiciones 

<img width="704" alt="image" src="https://github.com/kubernetes/website/assets/5762081/c547fe2c-bbec-41c8-bdf0-40eb80601f3b">

Reply:
@krol3 @ramrodo @92nqb quizá sería bueno discutir este punto la próxima reunión semanal

Reply:
Me parece que usar _"etiqueta"_ es lo adecuado. Aquí unos ejemplos de otras documentaciones:
- [Microsoft: "Usar etiquetas en un clúster de Azure Kubernetes Service (AKS)"](https://learn.microsoft.com/es-es/azure/aks/use-labels)
- [Google: "Crea y administra etiquetas de clúster y grupo de nodos"](https://cloud.google.com/kubernetes-engine/docs/how-to/creating-managing-labels?hl=es-419)